gazebo_ros/test/test_conversions.cpp

```diff
@@ -15,26 +15,47 @@
 #include <gazebo_ros/conversions.hpp>
 #include <gtest/gtest.h>
 
-TEST(TestConversions, RosVectorIgnVector)
+TEST(TestConversions, Vector3)
 {
-  // Ignition
-  ignition::math::Vector3d vec(1.0, 2.0, 3.0);
-
   // Ign to ROS
+  ignition::math::Vector3d vec(1.0, 2.0, 3.0);
   auto msg = gazebo_ros::Convert<geometry_msgs::msg::Vector3>(vec);
-
   EXPECT_EQ(1.0, msg.x);
   EXPECT_EQ(2.0, msg.y);
   EXPECT_EQ(3.0, msg.z);
-
   // ROS to Ign
   vec = gazebo_ros::Convert<ignition::math::Vector3d>(msg);
-
   EXPECT_EQ(1.0, vec.X());
   EXPECT_EQ(2.0, vec.Y());
   EXPECT_EQ(3.0, vec.Z());
 }
 
+TEST(TestConversions, Quaternion)
+{
+  // Ign to ROS
+  ignition::math::Quaterniond quat(1.0, 0.2, 0.4, 0.6);
+  auto quat_msg = gazebo_ros::Convert<geometry_msgs::msg::Quaternion>(quat);
+  EXPECT_EQ(0.2, quat_msg.x);
+  EXPECT_EQ(0.4, quat_msg.y);
+  EXPECT_EQ(0.6, quat_msg.z);
+  EXPECT_EQ(1.0, quat_msg.w);
+}
+
+TEST(TestConversions, Time)
+{
+  // Time conversions
+  // Gazebo to rclcpp
+  gazebo::common::Time time(200, 100);
+  auto rostime = gazebo_ros::Convert<rclcpp::Time>(time);
+  builtin_interfaces::msg::Time tmp(rostime);
+  EXPECT_EQ(200, tmp.sec);
+  EXPECT_EQ(100u, tmp.nanosec);
```

Comment:
Instead of converting to a message, could we directly check `rostime.nanoseconds()`?